.taskcluster.yml

```diff
@@ -161,7 +161,7 @@ tasks:
                cd /code &&
                git checkout ${head_rev} &&
                python http_service/download_models.py &&
-               taskboot --target /code build-compose --write /images"
+               taskboot --target /code build-compose --registry=registry.hub.docker.com --write /images"
```

Comment:
A thought for the future: it'd be nicer if the registry was specified when pushing (in theory you could push the same image to different registries).

Reply:
The thing is that `push-artifact` reads the full `RepoTags` written in the image, and use that for pushing (so you can push to multiple registries already).
So I set the registry during the build steps, instead of relying on multiple arguments at push to specify where to push what